docs/using/staking.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Staking
+authors: "Tim McMackin"
+last_update:
+  date: 17 December 2024
+---
+
+Staking is the process of temporarily locking tez on the Tezos platform.
```

Comment:
State right from the beginning the usefulness for yourself:
```suggestion
Staking is the process of temporarily locking tez on the Tezos platform in exchange of rewards.
```

Reply:
I made this "in exchange for rewards".